Src/Support/GoogleApis/Apis/[Media]/Upload/ResumableUpload.cs

```diff
@@ -522,15 +643,16 @@ protected virtual void ProcessResponse(HttpResponseMessage httpResponse)
                 }.CreateRequest();
 
             // Prepare next chunk to send.
-            if (StreamLength != UnknownSize)
+            int contentLength;

```

Comment:
Don't mind *too* much either way. Another alternative would be to create another whole method of `PrepareNextChunk` which did the right thing.

My main beef is having a local variable declaration without initialization. But let's not get too hung up on it.